modules/ROOT/pages/keyword-glossary.adoc

```diff
@@ -1034,6 +1034,9 @@ xref::functions/temporal/index.adoc#functions-temporal-truncate-overview[Truncat
 
 | xref:syntax/expressions.adoc#existential-subqueries[EXISTS {...}]
 | Evaluates the existence of a subquery.
+
+| xref:syntax/expressions.adoc#type-predicate-expressions[IS :: `type`]
+| Verifies that an expression has a certain type.
```

Comment:
Not sure if has is correct, would "Verifies that an expression is of a certain type" be better? Or is an instance of a certain type